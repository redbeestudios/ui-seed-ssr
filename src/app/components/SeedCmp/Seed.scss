@import "./../../assets/css/variables";

.seed {
  height: 180px;

  .seed__content {
    display: flex;
    flex-direction: column;
    align-items: center;
  }

  .seed__title {
    margin: 10px 0 0 0;
    font-weight: bold;
    font-size: 20pt;
  }

  .seed__message {
    margin: 5px 0 0 0;
    font-weight: bold;
    font-size: 15pt;
  }
}
