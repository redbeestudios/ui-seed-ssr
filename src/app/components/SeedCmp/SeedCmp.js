import React, { Component } from 'react';

import './Seed.scss'

const SeedCmp = ({greeting}) => {
    return (
        <div className='seed'>
          <div className='seed__content'>
            <img className='seed__img' src='/static/redbee-logo.svg' />
            <span className='seed__title'>redbee</span>
            <span className='seed__message'>{greeting}</span>
          </div>
        </div>
    );
}

export default SeedCmp;
