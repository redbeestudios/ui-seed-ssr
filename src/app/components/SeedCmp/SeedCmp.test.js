import React from "react";
import { shallow } from "enzyme";
import SeedCmp from "./SeedCmp";

describe("MyComponent", () => {
  it("should render correctly", () => {
    const message = "aMessage";
    const component = shallow(<SeedCmp greeting={message} />);
    expect(component).toMatchSnapshot();
    expect(component.find(".seed__message").text()).toEqual(message);
  });
});
