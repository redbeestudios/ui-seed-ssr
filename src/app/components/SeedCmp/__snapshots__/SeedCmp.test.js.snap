// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly 1`] = `
<div
  className="seed"
>
  <div
    className="seed__content"
  >
    <img
      alt="The redbee logo"
      className="seed__img"
      src="/static/redbee-logo.svg"
    />
    <span
      className="seed__title"
    >
      redbee
    </span>
    <span
      className="seed__message"
    >
      aMessage
    </span>
  </div>
</div>
`;
